app/components/views/GetStartedPage/index.js

```diff
@@ -19,7 +20,18 @@ class GetStartedPage extends React.Component {
   }
 
   componentDidMount() {
-    if (!this.props.isAdvancedDaemon) {
+    const {rpc_password, rpc_user, rpc_cert, rpc_host, rpc_port} = getRemoteCredentials();
+    const hasAllCredentials = rpc_password.length > 0 && rpc_user.length > 0 && rpc_cert.length > 0 && rpc_host.length > 0 && rpc_port.length > 0;
+    const hasAppData = getAppdataPath().length > 0;
+
+    if(hasAllCredentials && hasAppData)
+      this.props.setCredentialsAppdataError();
+
+    if (!this.props.openForm && hasAppData) {
+      this.props.onStartDaemon(null, getAppdataPath());
+    } else if (!this.props.openForm && hasAllCredentials) {
+      this.props.onStartDaemon(getRemoteCredentials());
+    } else if (!this.props.isAdvancedDaemon) {
```

Comment:
After doing some testing it's clear that this should be the first check done.  On !advanced reload it is using the previously entered rpc credentials I had entered.  I'd just put:

`if (!this.props.isAdvancedDaemon) { this.props.onStartDaemon(); break; }` at the top of componentDidMount()